src/contribute/index.md

```diff
@@ -174,6 +174,16 @@ across Java, Python, and Go, and every Beam runner.
  - JIRA: [sdk-go](https://issues.apache.org/jira/issues/?jql=project%20%3D%20BEAM%20AND%20component%20%3D%20sdk-go) / [BEAM-2083](https://issues.apache.org/jira/browse/BEAM-2083) |
  - Contact: [Henning Rohde](mailto:herohde@google.com)
 
+### Python 3 Support
```

Comment:
I would actually put this up higher. Many of the works in progress here are nearly abandoned and they should be more towards the bottom. At some point you or I or someone else should sort them.